addons/mail/controllers/discuss/channel.py

```diff
@@ -23,30 +23,30 @@ def _process_request_for_all(self, store, **kwargs):
 
 
 class ChannelController(http.Controller):
-    @http.route("/discuss/channel/members", methods=["POST"], type="json", auth="public", readonly=True)
+    @http.route("/discuss/channel/members", methods=["POST"], type='rpc', auth="public", readonly=True)
```

Comment:
Hello, thank you for the PR, this will remove a lot of confusion for newcomers!

Just a small request, could you please keep the quotes formatted as they are in our scope/in all modules?

Thanks!

Reply:
done!